instagram/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
```

Comment:
This is no longer necessary, as you are not using the React logo in your application